src/sentry/static/sentry/app/components/charts/styles.tsx

```diff
@@ -1,5 +1,8 @@
+import React from 'react';
```

Comment:
Feels a little weird having react in a styles file

Reply:
I'm not sure this component will be here for long. I want to consolidate similar ? icons in other parts of the application next.